ci-operator/step-registry/redhat-appstudio/rhtap-installer/install-rhtap/redhat-appstudio-rhtap-installer-install-rhtap-commands.sh

```diff
@@ -46,9 +50,27 @@ TPA__OIDC__TESTING_USER_CLIENT_SECRET="0e6bf990-43b4-4efb-95d7-b24f2b94a525" # n
 TPA__OIDC__WALKER_CLIENT_SECRET="5460cc91-4e20-4edd-881c-b15b169f8a79" # notsecret
 TPA__POSTGRES__POSTGRES_PASSWORD="postgres123456" # notsecret
 TPA__POSTGRES__TPA_PASSWORD="postgres1234" # notsecret
+OPENSHIFT_API="$(yq e '.clusters[0].cluster.server' $KUBECONFIG)"
+TAS__SECURESIGN__FULCIO__ORG_EMAIL='rhtap-qe-ci@redhat.com'
+TAS__SECURESIGN__FULCIO__ORG_NAME='RHTAP CI Jobs'
+TAS__SECURESIGN__FULCIO__OIDC__URL='http://localhost:3030'
+TAS__SECURESIGN__FULCIO__OIDC__CLIENT_ID="fake-one"
+TAS__SECURESIGN__FULCIO__OIDC__TYPE="dex"
+NAMESPACE=rhtap
 
+yq -i 'del(.clusters[].cluster.certificate-authority-data) | .clusters[].cluster.insecure-skip-tls-verify=true' $KUBECONFIG
+OPENSHIFT_PASSWORD="$(cat $KUBEADMIN_PASSWORD_FILE)"
 
-NAMESPACE=rhtap
+timeout --foreground 5m bash  <<- "EOF"
+    while ! oc login "$OPENSHIFT_API" -u kubeadmin -p "$OPENSHIFT_PASSWORD" --insecure-skip-tls-verify=true; do
```

Comment:
why do you need to login to the openshift? it already exported the KUBECONFIG by default.

Reply:
It is exported with old certificates and will not connect after adding the new ones

Reply:
get it. htanks